docs/creating-service-bindings/service-binding-resource.md

```diff
@@ -4,8 +4,122 @@ sidebar_position: 6
+| .spec.application | array (string) | It is used to uniquely identify an application resource |
+| .spec.application.group | string | It is the API group of the application resource |
+| .spec.application.version | string | It is the API group version of the application resource |
+| .spec.application.resource | string | It is the type of the application resource. It can be collected from `.kind` field of the application resource |
+| .spec.application.name | string | It is the unique name of the application. It can be collected from `.metadata.name` field of the application object |
+| .spec.service | array (string) | It is used to uniquely identify a service resource |
+| .spec.service.group | string | It is the API group of the service resource |
+| .spec.service.version | string | It is the API group version of the application resource |
+| .spec.service.resource | string | It is the type of the service resource. It can be collected from `.kind` field of the service resource |
+| .spec.service.name | string | It is the unique name of the service. It can be collected from `.metadata.name` field of the service object |
+
+## Service Binding Status
+
+`.status` describes current state of binding provided by the operator. The operator continually and actively manages every object's actual state to match the desired state.
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| .applications | string | It describes each bound application resource |
+| .conditions | string | It represents the latest available observations of Service Binding's state |
+| .secret | string | represents the name of the binding secret created by the Service Binding Operator |
+
+A sample service binding status:
+
+```yaml
+status:
+  applications:
+  - name: nodejs-app
+    group: apps
+    version: v1
+    resource: deployments
+  conditions:
+  - lastHeartbeatTime: "2020-10-15T13:23:36Z"
+    lastTransitionTime: "2020-10-15T13:23:23Z"
+    status: "True"
+    type: CollectionReady
+  - lastHeartbeatTime: "2020-10-15T13:23:36Z"
+    lastTransitionTime: "2020-10-15T13:23:23Z"
+    status: "True"
+    type: InjectionReady
+  - lastHeartbeatTime: "2020-10-15T13:23:36Z"
+    lastTransitionTime: "2020-10-15T13:23:23Z"
+    status: "True"
+    type: Ready
+  secret: binding-request-72ddc0c540ab3a290e138726940591debf14c581
+```
+
+### Conditions
+
+Service Binding Status Conditions have three types:
+
+| Condition | Description |
+| -------- | ----------- |
+| CollectionReady | It represents collection of secrets from the service. On failure of collecting secrets from the service, the `conditions.status` becomes "False" with a reason.
+ |
+| InjectionReady | It represents injection of secret into the application. On failure to inject information into the service, the `conditions.status` becomes "False" with a reason. |
+| Ready | It represents overall ready status. On failure, the `conditions.status` becomes "False" with a reason. |
```

Comment:
> It represents overall ready status. On failure, the `conditions.status` becomes "False" with a reason. |

How about: "Indicates if binding is successful. On failure, the condition.status is set to "False" and condition.reason provides more details."